<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>华文中宋</family>
    <pointsize>10</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:rgb(255, 255, 255);
	border-radius:20px;
	border:2px solid rgba(45, 45, 45, 0.5);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QToolButton{
	background-color: rgba(0, 0, 0,0);
	border: none;
}

QToolButton:hover{
	background-color:rgb(203, 203, 203);
}

QToolButton#exit_Button{
	border-image:url(:/img/X.svg);
}

QToolButton#minus_Button{
	border-image:url(:/img/Minus.svg);
}

QPushButton{
	background-color:rgba(0,0,0,0);
	border: none;
}

QPushButton:hover{
	background-color:rgb(203, 203, 203);
}

QLabel#date{
	background-color:rgba(255,255,255,0);
}

QWidget#TaskWidget{
	background-color:rgb(245, 245, 245);
	border-bottom-left-radius: 20px;
    border-bottom-right-radius:20px;
	border:2px solid rgba(45, 45,45, 0.5);
	border-top-color:rgba(203,203,203, 0.5);
}</string>
   </property>
   <widget class="QWidget" name="TaskWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>1293</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView{
	background-color:rgb(255, 255, 255);
	border-radius:20px;
	border: 1px solid rgba(204, 204, 204, 0.3);
}

QLabel#task1_label,QLabel#task2_label,QLabel#task3_label,QLabel#task4_label,QLabel#timeline_label{
	font: 700 15pt &quot;HONOR Sans Display GBK&quot;;
}
/*
QLabel#timeline_label{
	font: 700 12pt &quot;UD Digi Kyokasho NK-B&quot;;
}
*/

QWidget#task1_mainwidget,QWidget#task2_mainwidget,QWidget#task3_mainwidget,QWidget#task4_mainwidget{
	background-color:#ffffff;
}

QScrollArea{
	border:none;
}

QLabel#task1_edit,QLabel#task2_edit,QLabel#task3_edit,QLabel#task4_edit:hover{
	background-color:rgb(208, 208, 208);
}</string>
    </property>
    <widget class="QListWidget" name="timeline">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>20</y>
       <width>256</width>
       <height>601</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="task4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>320</y>
       <width>360</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="task1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>360</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="task3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>360</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="task2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>360</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="timeline_label">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>TimeLine</string>
     </property>
    </widget>
    <widget class="QLabel" name="task2_label">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>30</y>
       <width>299</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#timeline_label{
	
	font: 700 12pt &quot;UD Digi Kyokasho NK-B&quot;;
}</string>
     </property>
     <property name="text">
      <string>TASK2</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="task2_area">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>340</width>
       <height>217</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="task2_mainwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>217</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="task1_label">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>30</y>
       <width>299</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#timeline_label{
	
	font: 700 12pt &quot;UD Digi Kyokasho NK-B&quot;;
}</string>
     </property>
     <property name="text">
      <string>TASK1</string>
     </property>
    </widget>
    <widget class="QLabel" name="task3_label">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>330</y>
       <width>299</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#timeline_label{
	
	font: 700 12pt &quot;UD Digi Kyokasho NK-B&quot;;
}</string>
     </property>
     <property name="text">
      <string>TASK3</string>
     </property>
    </widget>
    <widget class="QLabel" name="task4_label">
     <property name="geometry">
      <rect>
       <x>513</x>
       <y>330</y>
       <width>299</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#timeline_label{
	
	font: 700 12pt &quot;UD Digi Kyokasho NK-B&quot;;
}</string>
     </property>
     <property name="text">
      <string>TASK4</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="task1_area">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>340</width>
       <height>217</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="task1_mainwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>217</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="task3_area">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>370</y>
       <width>340</width>
       <height>217</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="task3_mainwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>217</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="task4_area">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>370</y>
       <width>340</width>
       <height>217</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="task4_mainwidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>340</width>
        <height>217</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="timelineTask">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>70</y>
       <width>201</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#timelineTask{
	border:none;
	background-color:#ffffff;
}</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="timelineArea">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>70</y>
       <width>201</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:#ffffff;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="uselesswidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>531</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="task1_edit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/icons.qrc">
       <normaloff>:/img/Edit.svg</normaloff>:/img/Edit.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="task2_edit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>40</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/icons.qrc">
       <normaloff>:/img/Edit.svg</normaloff>:/img/Edit.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="task3_edit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>338</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/icons.qrc">
       <normaloff>:/img/Edit.svg</normaloff>:/img/Edit.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="task4_edit">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>338</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource/icons.qrc">
       <normaloff>:/img/Edit.svg</normaloff>:/img/Edit.svg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>216</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="topleft">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="plus">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/img/Plus.svg</normaloff>:/img/Plus.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calendar">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/img/Calendar.svg</normaloff>:/img/Calendar.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
         <hintingpreference>PreferDefaultHinting</hintingpreference>
        </font>
       </property>
       <property name="text">
        <string>2000/00/00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="topright">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="user">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource/icons.qrc">
         <normaloff>:/img/User.svg</normaloff>:/img/User.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="minus_Button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="exit_Button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>TaskWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resource/icons.qrc"/>
 </resources>
 <connections/>
</ui>
